apps/dashboard/Gemfile

```diff
@@ -8,7 +8,7 @@ gem 'sass-rails', '~> 5.0'
 # Use Uglifier as compressor for JavaScript assets
 gem 'uglifier', '>= 1.3.0'
 # Use CoffeeScript for .coffee assets and views
-gem 'coffee-rails', '~> 4.1.0'
+gem 'coffee-rails', '~> 4.2'
```

Comment:
Weren't changes to the Dashboard going to be reverted?

Reply:
I reverted the change to `font-awesome-sass` but not the others. Do I need to revert all Gemfile and Gemfile.lock changes for dashboard?

Reply:
I'm close to finishing the upgrade to Rails 5.2.3. I had to upgrade `coffee-rails` to make it work. I think that may  be the only conflict. I think that this will be a easy conflict to resolve, so never mind. I'm good.